lib/elocaps/tools/functions.dart

```diff
@@ -41,3 +42,44 @@ CapsMode stringToCapsMode(String mode) {
       return CapsMode.single;
   }
 }
+
+String scoreToWinOrLose(int score) {
+  if (score > 0) {
+    return ElocapsTextConstant.victory;
+  } else if (score == 0) {
+    return ElocapsTextConstant.draw;
+  } else {
+    return ElocapsTextConstant.defeat;
+  }
```

Comment:
You can shorten this by removing the else, since each case before returns something